zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -99,7 +99,7 @@ public Boolean call() {
             return doApproval();
         } catch (Exception e) {
```

Comment:
We are not using the exception that is thrown but showing our own message. Would it be worth us logging the exact exception details thrown in addition to what we have? (Definitely not a blocker, as this PR doesn't really change the logic here). Same in a few other places

Reply:
It's being added to the error log event on line 111 - its passed into `logException` - this adds the exception to the log event, sets the event message and logs it to std out.